sass/patternfly-react/_utilization-bar.scss

```diff
@@ -0,0 +1,10 @@
+strong .label-strong {
+  font-weight: 600;
+}
+span .display-inline-block {
+  display: inline-block;
+}
+div, span, strong .label-text {
+  font-size: 12px !important;
+  color: black;
+}
```

Comment:
I don't think we want to add these broad stroke selectors. These settings should be put on the classes for the particular objects they are being used for rather than adding new generic selectors here.

We are also trying to refrain from `~important` and use specificity instead (unless it is absolutely necessary to override a components `!important` from its custom css).

Reply:
I hope I finally got what you meant :D